GetRecd/Info.plist

```diff
@@ -2,19 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>UIAppFonts</key>
```

Comment:
Re-ordering the fonts here may lead to further merge conflicts down the line. Potentially keep as sprint has it?